ext/ext-api/api-qldb/src/main/resources/META-INF/dirigible/qldb/extensions/qldb.d.ts

```diff
@@ -0,0 +1,72 @@
+declare module "@dirigible/qldb" {
+    module QLDBRepository {
+        /**
+         *  Constructs a repository with given ledger and table.
+         *  Notice: If the table does not exist you must call .createTable()
+         */
+        constructor(ledgerName: string, tableName: string);
+
+        /**
+         *  Creates the repository table.
+         */
+        function createTable();
+
+        /**
+         *  Drops the repository table.
+         *  Notice: Dropping tables in QLDB inactivates them.
+         *  You can undo that with an UNDROP statement in PartiQL.
+         */
+        function dropTable();
+
+        /**
+         *  Inserts the entry in the repository.
+         *  On success returns the inserted entry.
+         */
+        function insert(entry: object): object;
```

Comment:
Why not generify the whole JS class? `d.ts` files allow you to declare classes as in TypeScript